### PR TITLE
Table: updates story from knobs to control

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
This PR updates story from knobs to control.
**Which issue(s) this PR fixes**:

Relates to #30182

### CODE REVIEW

**Comment by thisisobate** [comment]
@dprokop  Noticed the sorting both by field display name and descending doesn't work as expected. I think @kaydelaney tried fixing something similar in the past. Is this something we need to look into right away?

File: packages/grafana-ui/src/components/Table/Table.story.tsx [hunk 1]
```diff
@@ -99,21 +112,25 @@ const defaultThresholds: ThresholdsConfig = {
   mode: ThresholdsMode.Absolute,
 };
 
-export const Simple = () => {
+export const Simple: Story = (args) => {
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/Table/Table.story.tsx:115]: Maybe change to Basic as a convention?
```suggestion
export const Basic: Story = (args) => {
```

**Review by hugohaggmark — APPROVED** [review]
Looks good and works as expected 🎉  I wonder if we really need sortByDisplayName?
image

**Comment by thisisobate** [comment]
I think it's needed if a user wants to test the sorting capabilities of the Form component. But seems it doesn't work so well on Storybook, it makes total sense to disable the `sortByDisplayName` control @hugohaggmark

**Comment by hugohaggmark** [comment]
> @dprokop Noticed the sorting both by field display name and descending doesn't work as expected. I think @kaydelaney tried fixing something similar in the past. Is this something we need to look into right away?

If this is an issue with the Table component, I think you should create an issue it there isn't one already. Better to separate this PR from any changes needed in the Table component.

**Comment by thisisobate** [comment]
Alright 👍

**Comment by thisisobate** [comment]
just checked the table panel on Grafana...works as expected. I think I would have to disable the `descending` control too

**Comment by hugohaggmark** [comment]
@thisisobate merge with main to fix the failing backend tests

**Pull request merged**